lib/editor/components/pattern/PatternStopCard.js

```diff
@@ -301,6 +379,80 @@ class PatternStopContents extends Component<Props, State> {
             </FormGroup>
           </Col>
         </Row>
+        {/* Pickup and drop off type selectors */}
+        <Row>
+          <Col xs={6}>
+            <FormGroup
```

Comment:
Nitpick: I would try to refactor the common parts of each of these selectors.

Reply:
I have made the fields now more generic and used just one selector options. Let me know what you think.